kubetest/gke.go

```diff
@@ -330,11 +331,17 @@ func (g *gkeDeployer) Up() error {
 			}
 		}
 	}
-	// TODO(zmerlynn): The version should be plumbed through Extract
-	// or a separate flag rather than magic env variables.
-	if v := os.Getenv("CLUSTER_API_VERSION"); v != "" {
-		args = append(args, "--cluster-version="+v)
+
+	if *gkeReleaseChannel != "" {
+		args = append(args, "--release-channel="+*gkeReleaseChannel)
+	} else {
+		// TODO(zmerlynn): The version should be plumbed through Extract
```

Comment:
Hmm...so what do we specify in `--extract` with this changes?

Reply:
The commit message explain that. IIUC the idea use `--extract=ci/latest-1.14` which hopefully would be compatible enough with what is in the release channel.
This has two problems though:
 * the 1.14 can be not available outside of release channel at all. Then  `--extract=ci/latest-1.14` would fail.
 * manual tracking of what version actually is mapping to `--gke-release-channel=rapid` needs to be done. So when the version in channel changes from 1.14 to 1.15 we also update the `--extract` parameter. As with all manual changes it can be easily forgotten.

Given those two concerns I would suggest to change the approach to implement new `--extract` syntax (e.g `--extract=gke-channel-XXXX`) and have single source of truth both for cluster version and extracted binaries.

Reply:
nope, `--extract=gke-latest-1.14` would fail, but `--extract=ci/latest-1.14` would be ok, and in GKE we only need to get the test tarball from the right OSS version.

Reply:
will patch up --extract as well